stl/cpp_on_shore/roger_lv/src/task_2/my_test.h

```diff
@@ -0,0 +1,112 @@
+#include <chrono>
+#include <iostream>
+#include <vector>
+
+class Person {
+ private:
+  int age = 24;
+  std::string name = "Roger";
+
+ public:
+  Person(){};
+  Person(int x, std::string n) {
+    age = x + 14;
+    name = n;
+  }
+  int GetAge() { return age; }
+  std::string GetName() { return name; }
+};
+
+void TestPushAndEmplaceTime(std::vector<Person>& vc) {
+  auto start = std::chrono::steady_clock::now();
+  for (int i = 0; i < 100000; ++i) {
+    vc.push_back(Person(10, "roger"));
+  }
+  auto end = std::chrono::steady_clock::now();
+  std::cout << "Test the push time: "
+            << std::chrono::duration_cast<std::chrono::nanoseconds>(end - start).count()
+            << " ns" << std::endl;
+  start = std::chrono::steady_clock::now();
+  for (int i = 0; i < 100000; ++i) {
+    vc.emplace_back(Person(10, "roger"));
+  }
+  end = std::chrono::steady_clock::now();
+  std::cout << "Test the emplace time: "
+            << std::chrono::duration_cast<std::chrono::nanoseconds>(end - start).count()
+            << " ns" << std::endl;
+}
+
+void TestAboutAt(std::vector<Person>& vc) {
```

Comment:
may be google test frame is a good idea

Reply:
I'll learn it later, but now my os has some problems sry.